try:
    x=30/0
except Exception as error:
    print(error)
finally:
    print("finaly all ways run")
    